packages/adapter-chiselstrike/tests/basic.test.ts

```diff
@@ -0,0 +1,37 @@
+import { runBasicTests } from "@next-auth/adapter-test"
+import { ChiselStrikeAdapter, ChiselStrikeAuthFetcher } from "../src"
+
+const fetcher = new ChiselStrikeAuthFetcher('http://localhost:8080', '1234')
```

Comment:
I think for the sake of simplicity and more reliable tests, we should do fetch calls instead of this. aligns with my proposal to remove the class altogether.

Reply:
Or are you saying fetcher objects should become adapter objects instead?  That we can easily do.

Reply:
(And by fetcher objects, I don't primarily mean `fetcher.fetch()`.  It's more about, eg, `fetcher.filter()` or `fetcher.users()`).

Reply:
The Fetcher class is modelled on other adapters' Client class, though its name is perhaps suboptimal.